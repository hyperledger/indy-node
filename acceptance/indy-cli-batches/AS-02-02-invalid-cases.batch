#setup environment
- pool create AS-pool gen_txn_file=./pool_transactions_genesis
pool connect AS-pool
- wallet create AS-02-wallet key=testkey storage_config={"pool":"AS-pool"}
wallet open AS-02-wallet key=testkey
#node adding invalid cases:
#demote node as STEWARD of ANOTHER node (FAIL)
did use XhYtvJqezMUKfF6KVNaGmT
- ledger node target=6G9QhQa3HWjRKeRmEvEkLbWWf2t7cw6KLtafzi494G4G alias=Node6 services=
#demote node as TRUST_ANCHOR (FAIL)
did use Rhx2qwaeiGyhU9vn4ynHSS
- ledger node target=4Tn3wZMNCvhSTXPcLinQDnHyj56DTLQtL61ki4jo2Loc alias=Node5 services=
#demote node as IDENTITY_OWNER (FAIL)
did use 5qhSKMkpqui6taGLTecwqf
- ledger node target=4Tn3wZMNCvhSTXPcLinQDnHyj56DTLQtL61ki4jo2Loc alias=Node5 services=
#promote node as STEWARD of ANOTHER node (FAIL)
did use XhYtvJqezMUKfF6KVNaGmT
- ledger node target=6G9QhQa3HWjRKeRmEvEkLbWWf2t7cw6KLtafzi494G4G alias=Node6 services=VALIDATOR
#promote node as TRUST_ANCHOR (FAIL)
did use Rhx2qwaeiGyhU9vn4ynHSS
- ledger node target=6G9QhQa3HWjRKeRmEvEkLbWWf2t7cw6KLtafzi494G4G alias=Node6 services=VALIDATOR
#promote node as IDENTITY_OWNER (FAIL)
did use 5qhSKMkpqui6taGLTecwqf
- ledger node target=6G9QhQa3HWjRKeRmEvEkLbWWf2t7cw6KLtafzi494G4G alias=Node6 services=VALIDATOR