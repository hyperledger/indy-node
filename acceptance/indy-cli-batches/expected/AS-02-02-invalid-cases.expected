- pool create AS-pool gen_txn_file=./pool_transactions_genesis
[1;31mPool config "AS-pool" already exists[0m

pool connect AS-pool
[1;32mPool "AS-pool" has been connected[0m

- wallet create AS-02-wallet key=testkey storage_config={"pool":"AS-pool"}
[1;31mWallet "AS-02-wallet" already exists[0m

wallet open AS-02-wallet key=testkey
[1;32mWallet "AS-02-wallet" has been opened[0m

did use XhYtvJqezMUKfF6KVNaGmT
[1;32mDid "XhYtvJqezMUKfF6KVNaGmT" has been set as active[0m

- ledger node target=6G9QhQa3HWjRKeRmEvEkLbWWf2t7cw6KLtafzi494G4G alias=Node6 services=
[1;31mTransaction has been rejected: XhYtvJqezMUKfF6KVNaGmT is not a steward of node 6G9QhQa3HWjRKeRmEvEkLbWWf2t7cw6KLtafzi494G4G[0m

did use Rhx2qwaeiGyhU9vn4ynHSS
[1;32mDid "Rhx2qwaeiGyhU9vn4ynHSS" has been set as active[0m

- ledger node target=4Tn3wZMNCvhSTXPcLinQDnHyj56DTLQtL61ki4jo2Loc alias=Node5 services=
[1;31mTransaction has been rejected: Missing some of {'node_port', 'client_port', 'client_ip', 'alias', 'node_ip'}[0m

did use 5qhSKMkpqui6taGLTecwqf
[1;32mDid "5qhSKMkpqui6taGLTecwqf" has been set as active[0m

- ledger node target=4Tn3wZMNCvhSTXPcLinQDnHyj56DTLQtL61ki4jo2Loc alias=Node5 services=
[1;31mTransaction has been rejected: Missing some of {'client_ip', 'node_ip', 'client_port', 'node_port', 'alias'}[0m

did use XhYtvJqezMUKfF6KVNaGmT
[1;32mDid "XhYtvJqezMUKfF6KVNaGmT" has been set as active[0m

- ledger node target=6G9QhQa3HWjRKeRmEvEkLbWWf2t7cw6KLtafzi494G4G alias=Node6 services=VALIDATOR
[1;31mTransaction has been rejected: XhYtvJqezMUKfF6KVNaGmT is not a steward of node 6G9QhQa3HWjRKeRmEvEkLbWWf2t7cw6KLtafzi494G4G[0m

did use Rhx2qwaeiGyhU9vn4ynHSS
[1;32mDid "Rhx2qwaeiGyhU9vn4ynHSS" has been set as active[0m

- ledger node target=6G9QhQa3HWjRKeRmEvEkLbWWf2t7cw6KLtafzi494G4G alias=Node6 services=VALIDATOR
[1;31mTransaction has been rejected: Missing some of {'client_ip', 'node_ip', 'alias', 'node_port', 'client_port'}[0m

did use 5qhSKMkpqui6taGLTecwqf
[1;32mDid "5qhSKMkpqui6taGLTecwqf" has been set as active[0m

- ledger node target=6G9QhQa3HWjRKeRmEvEkLbWWf2t7cw6KLtafzi494G4G alias=Node6 services=VALIDATOR
[1;31mTransaction has been rejected: Missing some of {'node_port', 'client_port', 'client_ip', 'alias', 'node_ip'}[0m

[1;32mWallet "AS-02-wallet" has been closed[0m
[1;32mPool "AS-pool" has been connected[0m
[1;32mGoodbye...[0m
